Clarify 3d vehicle model sharing (#190)
FS-Online/Formula-Student-Driverless-Simulator: docs/import-car-3d-model.md

# 7. Selecting a car
Update your settings.json `PawnPaths -> DefaultCar -> PawnBP` to point at the newly created pawn. 
It should (tm) work.

# 8. Sharing your vehicle with the world!

> Wouldn't it be amazing if other people could enjoy your vehicle 3d model as well?

Reviewer: ```suggestion
Wouldn't it be amazing if other people could enjoy your team's vehicle 3d model as well?
```

> You are welcome to create a pull request with your vehicle and it might be added to the reop!

Reviewer: ```suggestion
You are welcome to create a pull request with your vehicle and it might be added to the base project repository!
```

> Contribution will be given in the readme of the project.

Reviewer: ```suggestion
Submit the new model as a pull request which is up to date with the current project state, with the vehicle model and pawn files added. Contribution will be given in the readme of the project.
```

> Please not that only GPLv2 licensed material is accepted into this repository.

Reviewer: ```suggestion
Please note that only GPLv2 licensed material is accepted into this repository.
```
